main/usbredir/APKBUILD

```diff
@@ -1,17 +1,15 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=usbredir
-pkgver=0.7.1
+pkgver=0.8.0
 pkgrel=0
 pkgdesc="USB network redirection protocol libraries"
 url="http://spice-space.org/page/UsbRedir"
 arch="all"
 options="!check"  # No test suite.
 license="GPL-2.0-or-later LGPL-2.1-or-later"
-depends_dev="libusb-dev"
```

Comment:
Why is `libusb-dev` being removed from `depends_dev` here?

Reply:
dependency tracing already finds pc:libusb-1.0 so it is redundant

Reply:
It's not unless `libusb-dev` is the package that provides `pc:libusb-1.0`.

Reply:
![image](https://user-images.githubusercontent.com/30738253/56841928-fd788e00-6866-11e9-98c0-240d2e18774c.png)

Reply:
I stand corrected.

```
# apk info --provides libusb-dev
libusb-dev-1.0.22-r0 provides:
pc:libusb-1.0=1.0.22
```